.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+# Overview
+
+# Checklist
+
+[ ] If I changed code, I ran `yarn build` and committed resulting changes.
+[ ] I added an example exercising this PRs functionality to `.github/workflows/test.yml` or explained why it doesn't make sense to do so.
```

Comment:
```suggestion
[ ] I added an example exercising this PRs functionality to `.github/workflows/test.yml` or explained why it doesn't make sense to do so.
```

[ ] I added an example exercising this PR's functionality to `.github/workflows/test.yml` or explained why it doesn't make sense to do so.